### DISCUSSION TITLE
Presentation slides

### DISCUSSION THREAD

**Discussion by shaido987** [PaddlePaddle/PaddleSpatial#299] [BaiduKDDCup2022] [answered]
Hello,

I saw that the presentation slides have not yet been added to the website. They would be helpful to get a quick overview of each solution. Any idea when they will be added?

**Comment by zhoujingbo** [comment, marked as answer]
We have updated the slides on the Baidu KDD Cup website for the ones we have got permission. Thanks ~ 

Regards
Jingbo

**Reply by shaido987** [reply]
I saw it. Thank you very much :)
